### ISSUE TITLE
Improper use of landmark

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1968]
## Bug Report

**Problematic behavior**
The `aria-label` attribute on the `region` landmark is redundant with the content of this section.
This is not an **important** navigation area.

**Expected behavior/code**
a p is sufficient to convey this textual information "Document public"

**Additional context/Screenshots**
<img width="791" height="197" alt="Image" src="https://github.com/user-attachments/assets/8d777b05-cb1e-4e17-9084-fc79b769b423" />

**Critères RGAA**
[12.6 Les zones de regroupement de contenus présentes dans plusieurs pages web (zones d’en-tête, de navigation principale, de contenu principal, de pied de page et de moteur de recherche) peuvent-elles être atteintes ou évitées ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#12.6)

**Impact**
In linear reading with screen readers, it will render "Document public, region - Document public" twice.
It's just a redundancy that can be avoided. It's a user experience improvement. Landmarks should be used appropriately. Too much ARIA kills ARIA.
